### ISSUE TITLE
[Request for comment] New chapter for document links(RFC, WebRTC ...)

### ISSUE THREAD

**Issue by hiroyuki-sato** [webrtc-for-the-curious/webrtc-for-the-curious#288]
Hello members.

What do you think to add these links to this book? (i.e: `13-references.md`)
I received [this comment](https://twitter.com/_pion/status/1541822739668951040?s=20&t=Q4WG_Zbjy6N6RSrsC-R-QQ) from Pion members.
(I said I want an RFC links page in WebRTC for curious)

I'm not familiar with WebRTC technologies. If someone tells me RFC documents(except introduced RFC in this book) for WebRTC, I'll add them.

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">I think it would be great to add a &#39;List of RFCs&#39; as another chapter :)<br><br>If you start it I am happy to translate to English.</p>&mdash; Pion (@_pion) <a href="https://twitter.com/_pion/status/1541822739668951040?ref_src=twsrc%5Etfw">June 28, 2022</a></blockquote>

This RFC list generates from [rfc-index](https://www.rfc-editor.org/rfc-index-100a.html). You can copy indexes from [this page](https://gist.githubusercontent.com/hiroyuki-sato/1ae6c968559ca0b0b83beaf907ed1a9e/raw/12725913791ff5a1e6042320948bdbd138cc3938/rfc_index.txt)
I referenced the WebRTC document links that [@voluntas](https://zenn.dev/voluntas/scraps/b776c706a1811a#comment-8ad9abe47088aa) created. Thanks!

**This list is still working in progress**

It needs a listing policy. For example,
* OpenSSL doesn't implement DTLS 1.3. But RFC6347 is already obsolete. 
* libwebrtc use `dcSCTP`. It violates RFC.
(Both examples were commented by @voluntas .)

If that knowledge gathers on this page, It will be very useful.

Original format.

```
Title of RFC Author list [ Month Year ] (Format) (Obsoletes xxxx) (Obsoleted-By xxxx)
(Updates xxxx) (Updated-By xxxx) (Also zzz##) (Status: status) (Stream: stream) (DOI: doi)
```

Current proposed format ( I removed `(Format)`, `(Stream: stream)` and `(DOI: doi)`

```
Title of RFC Author list [ Month Year ]  (Obsoletes xxxx) (Obsoleted-By xxxx)
(Updates xxxx) (Updated-By xxxx) (Also zzz##) (Status: status)
```

### DTLS

* [RFC6347](http://www.rfc-editor.org/info/rfc6347): Datagram Transport Layer Security Version 1.2 E. Rescorla, N. Modadugu [ January 2012 ]  (Obsoletes [RFC4347](http://www.rfc-editor.org/info/rfc4347)) (Obsoleted-By [RFC9147](http://www.rfc-editor.org/info/rfc9147)) (Updated-By [RFC7507](http://www.rfc-editor.org/info/rfc7507), [RFC7905](http://www.rfc-editor.org/info/rfc7905), [RFC8996](http://www.rfc-editor.org/info/rfc8996), [RFC9146](http://www.rfc-editor.org/info/rfc9146)) (Status: PROPOSED STANDARD)
* ~~[RFC9147](http://www.rfc-editor.org/info/rfc9147): The Datagram Transport Layer Security (DTLS) Protocol Version 1.3 E. Rescorla, H. Tschofenig, N. Modadugu [ April 2022 ]  (Obsoletes [RFC6347](http://www.rfc-editor.org/info/rfc6347)) (Status: PROPOSED STANDARD)~~ _OpenSSL doesn't implement DTLS 1.3. But RFC6347 is already obsolete._

### DataChannel

* [RFC8831](http://www.rfc-editor.org/info/rfc8831): WebRTC Data Channels R. Jesup, S. Loreto, M. Tüxen [ January 2021 ]  (Status: PROPOSED STANDARD)
* [RFC8832](http://www.rfc-editor.org/info/rfc8832): WebRTC Data Channel Establishment Protocol R. Jesup, S. Loreto, M. Tüxen [ January 2021 ]  (Status: PROPOSED STANDARD)
* [RFC8833](http://www.rfc-editor.org/info/rfc8833): Application-Layer Protocol Negotiation (ALPN) for WebRTC M. Thomson [ January 2021 ]  (Status: PROPOSED STANDARD)
* [RFC8834](http://www.rfc-editor.org/info/rfc8834): Media Transport and Use of RTP in WebRTC C. Perkins, M. Westerlund, J. Ott [ January 2021 ]  (Status: PROPOSED STANDARD)

### SCTP

* [RFC9260](http://www.rfc-editor.org/info/rfc9260): Stream Control Transmission Protocol R. Stewart, M. Tüxen, K. Nielsen [ June 2022 ]  (Obsoletes [RFC4460](http://www.rfc-editor.org/info/rfc4460), [RFC4960](http://www.rfc-editor.org/info/rfc4960), [RFC6096](http://www.rfc-editor.org/info/rfc6096), [RFC7053](http://www.rfc-editor.org/info/rfc7053), [RFC8540](http://www.rfc-editor.org/info/rfc8540)) (Status: PROPOSED STANDARD)
* [RFC8899](http://www.rfc-editor.org/info/rfc8899): Packetization Layer Path MTU Discovery for Datagram Transports G. Fairhurst, T. Jones, M. Tüxen, I. Rüngeler, T. Völker [ September 2020 ]  (Updates [RFC4821](http://www.rfc-editor.org/info/rfc4821), [RFC4960](http://www.rfc-editor.org/info/rfc4960), [RFC6951](http://www.rfc-editor.org/info/rfc6951), [RFC8085](http://www.rfc-editor.org/info/rfc8085), [RFC8261](http://www.rfc-editor.org/info/rfc8261)) (Status: PROPOSED STANDARD)
* [RFC8261](http://www.rfc-editor.org/info/rfc8261): Datagram Transport Layer Security (DTLS) Encapsulation of SCTP Packets M. Tuexen, R. Stewart, R. Jesup, S. Loreto [ November 2017 ]  (Updated-By [RFC8899](http://www.rfc-editor.org/info/rfc8899), [RFC8996](http://www.rfc-editor.org/info/rfc8996)) (Status: PROPOSED STANDARD)
* [RFC8260](http://www.rfc-editor.org/info/rfc8260): Stream Schedulers and User Message Interleaving for the Stream Control Transmission Protocol R. Stewart, M. Tuexen, S. Loreto, R. Seggelmann [ November 2017 ]  (Status: PROPOSED STANDARD)
* [RFC8841](http://www.rfc-editor.org/info/rfc8841): Session Description Protocol (SDP) Offer/Answer Procedures for Stream Control Transmission Protocol (SCTP) over Datagram Transport Layer Security (DTLS) Transport C. Holmberg, R. Shpount, S. Loreto, G. Camarillo [ January 2021 ]  (Status: PROPOSED STANDARD)
* [RFC7765](http://www.rfc-editor.org/info/rfc7765): TCP and Stream Control Transmission Protocol (SCTP) RTO Restart P. Hurtig, A. Brunstrom, A. Petlund, M. Welzl [ February 2016 ]  (Status: EXPERIMENTAL)
* [RFC6951](http://www.rfc-editor.org/info/rfc6951): UDP Encapsulation of Stream Control Transmission Protocol (SCTP) Packets for End-Host to End-Host Communication M. Tuexen, R. Stewart [ May 2013 ]  (Updated-By [RFC8899](http://www.rfc-editor.org/info/rfc8899)) (Status: PROPOSED STANDARD)
* [RFC6525](http://www.rfc-editor.org/info/rfc6525): Stream Control Transmission Protocol (SCTP) Stream Reconfiguration R. Stewart, M. Tuexen, P. Lei [ February 2012 ]  (Status: PROPOSED STANDARD)
* [RFC6083](http://www.rfc-editor.org/info/rfc6083): Datagram Transport Layer Security (DTLS) for Stream Control Transmission Protocol (SCTP) M. Tuexen, R. Seggelmann, E. Rescorla [ January 2011 ]  (Updated-By [RFC8996](http://www.rfc-editor.org/info/rfc8996)) (Status: PROPOSED STANDARD)
* [RFC5827](http://www.rfc-editor.org/info/rfc5827): Early Retransmit for TCP and Stream Control Transmission Protocol (SCTP) M. Allman, K. Avrachenkov, U. Ayesta, J. Blanton, P. Hurtig [ May 2010 ]  (Status: EXPERIMENTAL)
* [RFC5061](http://www.rfc-editor.org/info/rfc5061): Stream Control Transmission Protocol (SCTP) Dynamic Address Reconfiguration R. Stewart, Q. Xie, M. Tuexen, S. Maruyama, M. Kozuka [ September 2007 ]  (Status: PROPOSED STANDARD)
* [RFC3758](http://www.rfc-editor.org/info/rfc3758): Stream Control Transmission Protocol (SCTP) Partial Reliability Extension R. Stewart, M. Ramalho, Q. Xie, M. Tuexen, P. Conrad [ May 2004 ]  (Status: PROPOSED STANDARD)

### SDP

* [RFC8859](http://www.rfc-editor.org/info/rfc8859): A Framework for Session Description Protocol (SDP) Attributes When Multiplexing S. Nandakumar [ January 2021 ]  (Status: PROPOSED STANDARD)  
* [RFC8853](http://www.rfc-editor.org/info/rfc8853): Using Simulcast in Session Description Protocol (SDP) and RTP Sessions B. Burman, M. Westerlund, S. Nandakumar, M. Zanaty [ January 2021 ]  (Status: PROPOSED STANDARD)  
* [RFC8843](http://www.rfc-editor.org/info/rfc8843): Negotiating Media Multiplexing Using the Session Description Protocol (SDP) C. Holmberg, H. Alvestrand, C. Jennings [ January 2021 ]  (Obsoleted-By [RFC9143](http://www.rfc-editor.org/info/rfc9143)) (Updates [RFC3264](http://www.rfc-editor.org/info/rfc3264), [RFC5888](http://www.rfc-editor.org/info/rfc5888), [RFC7941](http://www.rfc-editor.org/info/rfc7941)) (Status: PROPOSED STANDARD)  

### RTP

* [RFC8852](http://www.rfc-editor.org/info/rfc8852): RTP Stream Identifier Source Description (SDES) A.B. Roach, S. Nandakumar, P. Thatcher [ January 2021 ]  (Status: PROPOSED STANDARD)
* [RFC7741](http://www.rfc-editor.org/info/rfc7741): RTP Payload Format for VP8 Video P. Westin, H. Lundin, M. Glover, J. Uberti, F. Galligan [ March 2016 ]  (Status: PROPOSED STANDARD)
* [RFC6904](http://www.rfc-editor.org/info/rfc6904): Encryption of Header Extensions in the Secure Real-time Transport Protocol (SRTP) J. Lennox [ April 2013 ]  (Updates [RFC3711](http://www.rfc-editor.org/info/rfc3711)) (Status: PROPOSED STANDARD)
* [RFC5764](http://www.rfc-editor.org/info/rfc5764): Datagram Transport Layer Security (DTLS) Extension to Establish Keys for the Secure Real-time Transport Protocol (SRTP) D. McGrew, E. Rescorla [ May 2010 ]  (Updated-By [RFC7983](http://www.rfc-editor.org/info/rfc7983)) (Status: PROPOSED STANDARD)
* [RFC4585](http://www.rfc-editor.org/info/rfc4585): Extended RTP Profile for Real-time Transport Control Protocol (RTCP)-Based Feedback (RTP/AVPF) J. Ott, S. Wenger, N. Sato, C. Burmeister, J. Rey [ July 2006 ]  (Updated-By [RFC5506](http://www.rfc-editor.org/info/rfc5506), [RFC8108](http://www.rfc-editor.org/info/rfc8108)) (Status: PROPOSED STANDARD)
* [RFC3711](http://www.rfc-editor.org/info/rfc3711): The Secure Real-time Transport Protocol (SRTP) M. Baugher, D. McGrew, M. Naslund, E. Carrara, K. Norrman [ March 2004 ]  (Updated-By [RFC5506](http://www.rfc-editor.org/info/rfc5506), [RFC6904](http://www.rfc-editor.org/info/rfc6904)) (Status: PROPOSED STANDARD)
* [RFC3611](http://www.rfc-editor.org/info/rfc3611): RTP Control Protocol Extended Reports (RTCP XR) T. Friedman, R. Caceres, A. Clark [ November 2003 ]  (Status: PROPOSED STANDARD)
* [RFC3550](http://www.rfc-editor.org/info/rfc3550): RTP: A Transport Protocol for Real-Time Applications H. Schulzrinne, S. Casner, R. Frederick, V. Jacobson [ July 2003 ]  (Obsoletes [RFC1889](http://www.rfc-editor.org/info/rfc1889)) (Updated-By [RFC5506](http://www.rfc-editor.org/info/rfc5506), [RFC5761](http://www.rfc-editor.org/info/rfc5761), [RFC6051](http://www.rfc-editor.org/info/rfc6051), [RFC6222](http://www.rfc-editor.org/info/rfc6222), [RFC7022](http://www.rfc-editor.org/info/rfc7022), [RFC7160](http://www.rfc-editor.org/info/rfc7160), [RFC7164](http://www.rfc-editor.org/info/rfc7164), [RFC8083](http://www.rfc-editor.org/info/rfc8083), [RFC8108](http://www.rfc-editor.org/info/rfc8108), [RFC8860](http://www.rfc-editor.org/info/rfc8860)) (Also [STD0064](http://www.rfc-editor.org/info/std0064)) (Status: INTERNET STANDARD)

### ICE/TURN/STUN

* [RFC8445](http://www.rfc-editor.org/info/rfc8445): Interactive Connectivity Establishment (ICE): A Protocol for Network Address Translator (NAT) Traversal A. Keranen, C. Holmberg, J. Rosenberg [ July 2018 ]  (Obsoletes [RFC5245](http://www.rfc-editor.org/info/rfc5245)) (Updated-By [RFC8863](http://www.rfc-editor.org/info/rfc8863)) (Status: PROPOSED STANDARD)
* [RFC8489](http://www.rfc-editor.org/info/rfc8489): Session Traversal Utilities for NAT (STUN) M. Petit-Huguenin, G. Salgueiro, J. Rosenberg, D. Wing, R. Mahy, P. Matthews [ February 2020 ]  (Obsoletes [RFC5389](http://www.rfc-editor.org/info/rfc5389)) (Status: PROPOSED STANDARD)
* [RFC8656](http://www.rfc-editor.org/info/rfc8656): Traversal Using Relays around NAT (TURN): Relay Extensions to Session Traversal Utilities for NAT (STUN) T. Reddy, A. Johnston, P. Matthews, J. Rosenberg [ February 2020 ]  (Obsoletes [RFC5766](http://www.rfc-editor.org/info/rfc5766), [RFC6156](http://www.rfc-editor.org/info/rfc6156)) (Status: PROPOSED STANDARD)

**Comment by mogren** [comment]
This is awesome, yes please! 😄 

My only nit pick would be to drop the spaces around the dates, so `[ February 2020 ]` -> `[February 2020]`.

**Comment by hiroyuki-sato** [comment]
Implemented #289.
